extension/parquet/parquet_statistics.cpp

```diff
@@ -350,6 +350,9 @@ unique_ptr<BaseStatistics> ParquetStatisticsUtils::TransformColumnStatistics(con
 			row_group_stats->Set(StatsInfo::CAN_HAVE_NULL_AND_VALID_VALUES);
 		}
 		return row_group_stats;
+	} else if (schema.schema_type == ParquetColumnSchemaType::VARIANT) {
+		//! FIXME: there are situations where VARIANT columns can have stats
```

Comment:
This is true for shredded Variants, when all values in the Variant column are shredded.
Leaving this FIXME as a reminder to look at this when we implement support for shredded Variants in a follow-up PR